lib/wptool.js

```diff
@@ -260,7 +260,7 @@ function nativeEnumerate(wpsdk, options, next) {
 			} else {
 				var emulators = parseAppDeployCmdListing(out, wpsdk);
 				next(null, {
-					devices: [{name: 'Device', udid: 0, index: 0, wpsdk: null}],
+					devices: [],
```

Comment:
OK, I just verified that we can't do this. If you only have WP SDK 8.1 installed, then the following code:

```javascript
var windowslib = require('./index');

windowslib.device.detect(function (err, results) {
    if (err) {
        console.error('ERROR', err);
    } else {
        console.info(results);
    }
});
```

outputs:

```
{ devices: [], issues: [] }
```

This is not correct.

Older versions of Windows Phone SDK are not able to detect if a device is connected and so it always reports a device. We must continue to do the same.

I installed the Windows 10 RTM and it lists a Windows mobile device as expected, however the uuid is not compatible when installing using the WP SDK 8.1 and older tooling.

I correct thing to do is to combine "legacy" devices from older WP SDKs with real devices from new WP SDKs. So, if we detect WP SDK 8.1 or older, then we have to inject this pseudo device entry. If we detect Windows 10 RTM or newer, then add the real devices. The legacy device name should be renamed from `Device` to `Window Phone 8.x Device` or something.

The correct output when Windows Phone SDK 8 and Windows 10 RTM are installed should be:

```
{ devices:
   [ { name: 'Device', udid: 0, index: 0, version: '6.3', wpsdk: '8.1' },
     { name: 'Lumia 1520 (RM-940)',
       udid: '00000015-ba33-30a8-0000-000000000000',
       index: 0,
       wpsdk: null,
       ip: '127.0.0.1' } ],
  issues: [] }
```